### PR TITLE
loaded pootle translations for french, arabic and spanish

### PR DESCRIPTION
**Pull request by g1039**

### CODE REVIEW

**Comment by g1039** [comment]
@miltontony Please review cc @saeedmb8

**Comment by g1039** [comment]
@saeedmb8 Ready for review

File: springboard_iogt/locale/ara_AE/LC_MESSAGES/messages.po [hunk 1]
```diff
@@ -26,7 +25,7 @@ msgstr ""
 
 #: springboard_iogt/utils.py:20
 msgid "Become a U-Reporter & start sharing!"
-msgstr ""
+msgstr "Voice matters"
```
Comment (smarzban) [inline springboard_iogt/locale/ara_AE/LC_MESSAGES/messages.po]: @TrevorOctober this doesn't look like the right Arabic translation for `Become a U-Reporter & start sharing!`

File: springboard_iogt/locale/ara_AE/LC_MESSAGES/messages.po [hunk 2]
```diff
@@ -155,6 +154,7 @@ msgid "Child"
 msgstr "الطفل"
 
 #: springboard_iogt/templates/personae.jinja2:10
+#, fuzzy
```
Comment (smarzban) [inline springboard_iogt/locale/ara_AE/LC_MESSAGES/messages.po]: @TrevorOctober please remove the `fuzzy`

**Comment by smarzban** [comment]
@TrevorOctober left some comments, can you also add some screenshots for these three languages that you added

File: springboard_iogt/locale/ara_AE/LC_MESSAGES/messages.po (hunk 1 above)
Comment (g1039) [inline springboard_iogt/locale/ara_AE/LC_MESSAGES/messages.po]: @saeedmb8 I will ask @Botswang for translation.

**Comment by g1039** [comment]
screen shot 2016-08-08 at 11 59 07 am
Spanish

**Comment by g1039** [comment]
screen shot 2016-08-08 at 11 18 30 am
French

**Comment by g1039** [comment]
screen shot 2016-08-08 at 11 17 35 am
Arabic

**Comment by g1039** [comment]
@saeedmb8 Please review

**Comment by smarzban** [comment]
👍

**Pull request merged**